lib/money-heuristics/analyzer.rb

```diff
@@ -1,21 +1,26 @@
 module MoneyHeuristics
   class Analyzer
     attr_reader :search_tree, :words
-    attr_accessor :str, :currencies
+    attr_accessor :str, :currencies, :methods
 
     def initialize(str, search_tree)
       @str = (str || '').dup
       @search_tree = search_tree
       @currencies = []
+      @methods = { iso_code: "search_by_iso_code", 
+                   symbol: "search_by_symbol", 
+                   name: "search_by_name" }
```

Comment:
Since these are basically method references it would make sense to turn them into symbols